test/integration_test/migration_stork_failures_test.go

```diff
@@ -0,0 +1,197 @@
+//go:build integrationtest
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/portworx/sched-ops/k8s/core"
+	storkops "github.com/portworx/sched-ops/k8s/stork"
+	"github.com/portworx/sched-ops/task"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	migrationAppKey = "mysql-1-pvc"
+	migrationKey    = "mysql-migration"
+	nsKey           = "delete-stork-pods"
+	storkNamespace  = "kube-system"
+)
+
+var storkLabel = map[string]string{"name": "stork"}
+
+func TestMigrationStorkFailures(t *testing.T) {
+	// reset mock time before running any tests
+	err := setMockTime(nil)
+	require.NoError(t, err, "Error resetting mock time")
+
+	logrus.Infof("Using stork volume driver: %s", volumeDriverName)
+	logrus.Infof("Backup path being used: %s", backupLocationPath)
+
+	t.Run("deleteStorkPodsSourceDuringMigrationTest", deleteStorkPodsSourceDuringMigrationTest)
+	t.Run("deleteStorkPodsDestDuringMigrationTest", deleteStorkPodsDestDuringMigrationTest)
```

Comment:
Can the common code from these 2 tests be combined, and use a small section that differentiates deleting src/dest?

Reply:
Good idea, created a common function to run both tests.